### ISSUE TITLE
podspec source and source_files is not taking the local path

### ISSUE THREAD

**Issue by sunil-chayagol** [CocoaPods/cocoapods-packager#218]
[!] Failed to load 'MyFramework' podspec: 
[!] Invalid `MyFramework.podspec` file: undefined method `scource_files=' for #<Pod::Specification name="MyFramework">
Did you mean?  source_files=.

 #  from /Users/sunil/workspace/karTesting/MyFramework/MyFramework.podspec:12
 #  -------------------------------------------
 #  s.source = {:path  => "Users/sunil/workspace/karTesting/MyFramework"}
 >  s.scource_files = "Users/sunil/workspace/karTesting/**/*.swift"
 #  end
 #  --

is it compulsory to host the framework in github cant we host locally .

**Comment by dnkoutso** [comment]
you have a typo `scource_files` should be `source_files`

**Comment by sunil-chayagol** [comment]
yes saw it after posting the issue. pod gets installed but the swift files are not copied.
pod 'MyFramework', :path => '/Users/sunil/workspace/karTesting/MyFramework'
this is the line i use in the podfile
